python/lsst/pipe/tasks/calibrateImage.py

```diff
@@ -324,6 +324,20 @@ class CalibrateImageConfig(pipeBase.PipelineTaskConfig, pipelineConnections=Cali
         target=lsst.meas.algorithms.SubtractBackgroundTask,
         doc="Task to perform final background subtraction, just before photoCal.",
     )
+    star_background_max_pedestal = pexConfig.Field(
+        dtype=float,
+        default=3.0,
+        doc="Maximum pedestal allowable in star_background measurement. If the computed value "
+            "exceeds this value, the background pedestal will be set to this value.",
```

Comment:
What are the units?  Just whatever the post_isr units are?

Reply:
Yeah, I didn't want to put units that for some reason may be different than our standard processing...